src/strings.js

```diff
@@ -1,25 +1,25 @@
-const sayHello = (string) => {
-  // your code here
+const sayHello = (world) => {
+  return  'Hello, ' + world + '!'; 
 };
 
-const uppercase = (string) => {
-  // your code here
+const uppercase = (abc) => {
```

Comment:
In this case changing the variable label on the function from `string` to `abc` makes it less explicit, the same is applicable to the remaining functions on this file.

It's worth noting that what goes between the parenthesis in the function definition is a placeholder label for the variable the function is going to receive when it's called. More at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions